### PR TITLE
Yelp search

### LINKED ISSUE #34: api call to backend yelp for search
Get yelp info from backend to populate page

### PR DESCRIPTION
**Pull request by zachjamesgreen**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

Issue Number: 
closes #34
can search with yelp

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by markcyen — APPROVED** [review]
A-O-K!

File: src/components/yelp_search.js
Comment (mkrumholz) [inline src/components/yelp_search.js:53]: Cool, yep! Good to see this getting data from the backend.

**Review by mkrumholz — APPROVED** [review]
Ok, cool. This looks good to me, but I wasn't able to test locally yet. Approving so we can get this onto main for major testing

**Comment by mkrumholz** [comment]
@zachjamesgreen looks like tests are failing here for circle so we should check that before merging.

File: src/components/cocktail.js
Comment (redferret) [inline src/components/cocktail.js:55]: I'll pin this form for later but it breaks the React frame work. It's easier this way but it's not something you would see in a React app. Sorta the downside of React. It flys in the face of reason in parts like this.

File: src/components/search.js
Comment (redferret) [inline src/components/search.js:19]: that was the right choice here. 
You're fighting the component lifecycle doing this and in turn breaking the React framework mentioned above.

**Review by redferret — APPROVED** [review]
loads of refactoring needs to be done later. Keep the idea of what the instructors would say about creating a Rails app you can do technically whatever but there are conventional patterns and convention would say this is not React.

**Pull request merged**
